src/btree/bt_debug.c

```diff
@@ -1322,6 +1322,30 @@ __debug_update(WT_DBG *ds, WT_UPDATE *upd, bool hexbyte)
     return (0);
 }
 
+/*
+ * __debug_ref_state --
+ *     Return a string representing the WT_REF state.
+ */
+static const char *
+__debug_ref_state(u_int state)
+{
+    switch (state) {
+    case WT_REF_DISK:
+        return ("disk");
+    case WT_REF_DELETED:
+        return ("deleted");
```

Comment:
If we keep this, alphabetize this!

Reply:
It's in flag declared order, not alphabetized order.